app/src/main/java/org/p2p/wallet/push_notifications/ineractor/PushNotificationsInteractor.kt

```diff
@@ -3,22 +3,28 @@ package org.p2p.wallet.push_notifications.ineractor
 import android.content.SharedPreferences
 import android.os.Build
 import androidx.core.content.edit
+import kotlinx.coroutines.delay
+import kotlinx.coroutines.launch
+import org.p2p.wallet.common.di.AppScope
+import org.p2p.wallet.infrastructure.network.data.ServerException
 import org.p2p.wallet.infrastructure.network.provider.TokenKeyProvider
 import org.p2p.wallet.push_notifications.model.DeviceInfo
 import org.p2p.wallet.push_notifications.model.DeviceToken
 import org.p2p.wallet.push_notifications.repository.DeviceTokenRepository
 import org.p2p.wallet.push_notifications.repository.PushTokenRepository
 
 private const val KEY_DEVICE_TOKEN = "KEY_DEVICE_TOKEN"
+private const val TOKEN_SEND_RETRY_DELAY = 60000L // ms
 
 class PushNotificationsInteractor(
     private val deviceTokenRepository: DeviceTokenRepository,
     private val pushTokenRepository: PushTokenRepository,
     private val tokenKeyProvider: TokenKeyProvider,
     private val sharedPreferences: SharedPreferences,
+    private val appScope: AppScope
 ) {
 
-    suspend fun updateDeviceToken(): DeviceToken {
+    suspend fun updateDeviceToken(retries: Int = 1) {
```

Comment:
i'd prefer to move 1 to constant and use/reuse it

Reply:
It's a one use value, so it's no need for constant

Reply:
Agree with @kabramovp2p, move to variable or to constant

Reply:
Okay then